lib/server.js

```diff
@@ -1,21 +1,29 @@
 "use strict";
 
-//Modules required
-var Hapi = require("hapi");
+var Hapi              = require("hapi");
+var Bluebird          = require("bluebird");
+var MongoMapper       = require("genesis").MongoMapper;
+var DatabaseConnector = require("./services/DatabaseConnector");
 
-//Setting the port for the application
-var port = process.env.PORT || 8080;
+var databaseConnector = DatabaseConnector.connect(process.env.DB_URL, process.env.DB_USER, process.env.DB_PASSWORD);
+var mapper            = new MongoMapper(databaseConnector);
```

Comment:
Additionally, what do you think should happen if there are no `DB_URL`, `DB_USER`, or `DB_PASSWORD` variables defined? Right now, the server crashes.